shared/build.gradle.kts

```diff
@@ -6,22 +6,20 @@ plugins {
 	alias(libs.plugins.kotest)
 	alias(libs.plugins.kover)
 	alias(libs.plugins.resources)
-	alias(libs.plugins.cocoapods)
```

Comment:
It's not needed at this stage so I decided to drop it to reduce number of dependencies